cypress/integration/jsEmployeesTableDOM.spec.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+Cypress.Commands.add('fillTable',
+  (fullName, position, city, age, salary) => {
+    cy.get('[data-qa="name"]').type(fullName);
+    cy.get('[data-qa="position"]').type(position);
+    cy.get('[data-qa="city"]').select(city);
+    cy.get('[data-qa="age"]').type(age);
+    cy.get('[data-qa="salary"]').type(salary);
+    cy.get('button').contains('Save to table').click();
+  });
+
+Cypress.Commands.add('checkData',
```

Comment:
`checkData` is too generic. Function name should explain what exactly we are checking for?